auth/src/main/java/org/aerogear/auth/AuthServiceConfig.java

```diff
@@ -1,8 +1,15 @@
 package org.aerogear.auth;
 
+import org.json.JSONObject;
+
 /**
  * Authentication service configuration
  */
 public class AuthServiceConfig {
     // TODO: to be implemented.
+
+    public JSONObject toJSON() {
+        // TODO: to be implemented.
+        return null;
```

Comment:
Please, throw an IllegalStateException("Not implemented") here instead of returning null.